examples/preprocessing/plot_muscle_detection.py

```diff
@@ -0,0 +1,84 @@
+"""
+===========================
+Annotate muscle artifacts
+===========================
+
+Muscle contractions produce high frequency activity that can mask brain signal
+of interest. Muscle artifacts can be produced when clenching the jaw,
+swallowing, or twitching a head muscle. Muscle artifacts are most notable
+in the range of 110-140 Hz.
+
+This example uses :func:`~mne.preprocessing.annotate_muscle_zscore` to annotate
+segments where muscle activity is likely present. This is done by band-pass
+filtering the data in the 110-140 Hz range. Then, the envelope is taken using
+the hilbert analytical signal to only consider the absolute amplitude and not
+the phase of the high frequency signal. The envelope is z-scored and summed
+across channels and divedid by the square root of the channel numer. To remove
+noisy transient peaks (muscle artifacts last several hundred milliseconds),
+the channel averaged z-scored is low-pass filtered to 4 Hz. Segments above a
```

Comment:
```suggestion
across channels and divided by the square root of the number of channels.
Because muscle artifacts last several hundred milliseconds, we then low-pass
filter the averaged z-scores at 4 Hz, to remove transient peaks. Segments above
a
```